docs/Blog-Post.md

```diff
@@ -0,0 +1,525 @@
+# ---
```

Comment:
Assuming there will be multiple blog posts in the future, perhaps we should rename this file?

Reply:
yeah sorry. it was originally announcement-post but i renamed it so you'd get the entire post instead of a small diff

Reply:
i'll rename right before mereg